packages/analytics/dashboard-renderer/src/components/DashboardTile.vue

```diff
@@ -40,6 +114,36 @@ const emit = defineEmits<{
   (e: 'edit-tile', tile: TileDefinition): void
 }>()
 
+const queryBridge: AnalyticsBridge | undefined = inject(INJECT_QUERY_PROVIDER)
+const { evaluateFeatureFlag } = composables.useEvaluateFeatureFlag()
+const { i18n } = composables.useI18n()
+const hasKebabMenuAccess = evaluateFeatureFlag('ma-3043-analytics-chart-kebab-menu', false)
+
+const chartData = ref<ExploreResultV4>()
+const exportModalVisible = ref<boolean>(false)
+const showKebabMenu = ref(false)
+
+const exploreLink = computed(() => {
+  if (queryBridge && queryBridge.exploreBaseUrl) {
+    const exploreQuery: ExploreQuery | AiExploreQuery = {
+      filters: [...props.context.filters, ...props.definition.query.filters ?? []],
+      metrics: props.definition.query.metrics as ExploreAggregations[] | AiExploreAggregations[] ?? [],
+      dimensions: props.definition.query.dimensions as QueryableExploreDimensions[] | QueryableAiExploreDimensions[] ?? [],
+      time_range: props.definition.query.time_range as TimeRangeV4 || props.context.timeSpec,
+
+    } as ExploreQuery | AiExploreQuery
+    // Explore only supports advanced or ai
+    const datasource = ['advanced', 'ai'].includes(props.definition.query.datasource) ? props.definition.query.datasource : 'advanced'
+    return `${queryBridge.exploreBaseUrl()}?q=${JSON.stringify(exploreQuery)}&d=${datasource}&c=${props.definition.chart.type}`
+  }
+
+  return ''
+})
+
+const csvFilename = computed<string>(() => i18n.t('csvExport.defaultFilename'))
+const canShowKebabMenu = computed(() => hasKebabMenuAccess && !['golden_signals', 'top_n', 'gauge'].includes(props.definition.chart.type))
+const showDescriptionInHeader = computed(() => ['top_n', 'golden_signals'].includes(props.definition.chart.type))
```

Comment:
Instead of special-casing this, would it be possible to only show the description if it's present?  Or would that break something I don't see?

Reply:
done

Reply:
I don't see any change?